Dockerfile.test.cpu

```diff
@@ -27,6 +27,13 @@ RUN apt-get install -y --no-install-recommends \
         build-essential \
         g++-4.8
 
+# install g++-7
+RUN apt-get install -y --no-install-recommends software-properties-common
+RUN apt-get update
+RUN add-apt-repository ppa:ubuntu-toolchain-r/test
+RUN apt-get update
+RUN apt-get install -y --no-install-recommends gcc-7 g++-7
```

Comment:
We can just say `g++-7`, it will bring `gcc-7`.

Reply:
Ok.